### DISCUSSION TITLE
How to enable AI Suggestions

### DISCUSSION THREAD

**Discussion by leonarhv** [drodil/backstage-plugin-qeta#238] [Q&A] [answered]
I'm making an AI module, but as I add It to my backstage installation, I can't see the AI suggestion on questions. Does this have any pre-requisites?

Maybe another plugin to add, or a config on app-config

**Comment by drodil** [comment, marked as answer]
Hi, did you check out the documentation: https://github.com/drodil/backstage-plugin-qeta/blob/main/docs%2Fai.md ?

You will need to build your own backend module for it or use the openai module in this repo

**Reply by leonarhv** [reply]
I did, but It was not loading the AI block on the frontend. After updating some libs everything worked, This question is already closed.

Thanks for answering anyways!!
